examples/src/test/scala/docs/kafka/KafkaDocExample.scala

```diff
@@ -111,16 +112,23 @@ object KafkaDocExample {
   //#wordPublisher
 
   val config: Config = ConfigFactory.parseString("""
-    akka.projection.slick = {
-
-      profile = "slick.jdbc.H2Profile$"
-
-      db = {
-       url = "jdbc:h2:mem:test1"
-       driver = org.h2.Driver
-       connectionPool = disabled
-       keepAliveConnection = true
+    akka.projection.jdbc = {
+      dialect = "h2-dialect"
+      blocking-jdbc-dispatcher {
+        type = Dispatcher
+        executor = "thread-pool-executor"
+        thread-pool-executor {
+          fixed-pool-size = 10
+        }
+        throughput = 1
       }
+    
+      offset-store {
+        schema = ""
+        table = "AKKA_PROJECTION_OFFSET_STORE"
+      }
+    
+      debug.verbose-offset-store-logging = false
```

Comment:
This was in need of some love, we moved to JDBC, but we still had some Slick settings. 

When running the main function, nothing was working.

Reply:
I'm not sure if this code was ever run, it was added for docs purposes. We should probably adapt it so it can be run in a test, and serve as a complete standalone example.

Reply:
Well, I run it now. ;-)

Agree that we can run it on tests. We have the Slick / Kafka integration one, but nothing for JDBC.